web/packages/teleport/src/components/ButtonLockedFeature/ButtonLockedFeature.tsx

```diff
@@ -29,17 +30,24 @@ export type Props = {
   [index: string]: any;
 };
 
-const salesUrl = 'https://goteleport.com/signup/enterprise/';
+const salesUrl = 'https://goteleport.com/r/upgrade-team';
```

Comment:
think we prefer screaming snake casing for js consts

```suggestion
const SALES_URL = 'https://goteleport.com/r/upgrade-team';
```